### PR TITLE
feat: Add README

### PR DESCRIPTION
**Pull request by PAndaContron**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ * Passionfruit
+ * Peach
+ * Pineapple
+ * Raspberry
+ * Strawberry
+ * Tomato
+
+### To be added:
+ * Watermelon
+
```
Comment (syntaxi) [inline README.md]: This section should probably be removed. Someone looking at the readme doesn't need to know that a fruit still needs to be added

File: README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# AdditionalFruits
+A set of fruits that grow using the SimpleFarming module. For more information, check out the [SimpleFarming Module](https://github.com/Terasology/SimpleFarming).
+
```
Comment (syntaxi) [inline README.md]: Nice link to the SimpleFarming repo but can you give a slightly bigger summary of the module. Emphasise that this repo contains fruits and that there are more plants in the  AdditionalVegetables and CommonEdibleFlora repos.

. Would be good if you could add an image in here, perhaps just a handful of the bushes in a row would be fine.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+ * Gooseberry
+ * Guava
+ * Melon
+ * Passionfruit
+ * Peach
+ * Pineapple
+ * Raspberry
+ * Strawberry
+ * Tomato
+
```
Comment (syntaxi) [inline README.md:17]: A contributing section would be nice as well. Doesn't need to be long, just mentioning that this module is for fruits, a link to SimpleFarming readme on how to add a plant and telling the reader to open a pull request with the new content.

**Review by syntaxi — DISMISSED** [review]
Only three changes, otherwise good. Once those are fixed up I'll approve the task :)

File: README.md (hunk 2 above)
Comment (PAndaContron) [reply README.md]: I can't find the CommonEdibleFlora repo.

**Review by skaldarnar — APPROVED** [review]

**Pull request merged**